// pipes-module/__ivy_ngcc__/fesm2015/pipes-module.js
import { NgModule } from '@angular/core';
import { Nl2BrPipeModule, Nl2BrPipe } from 'nl2br-pipe';
import { TrustHtmlModule, TrustHtmlPipe } from 'trust-html';

/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import * as ɵngcc0 from '@angular/core';
class PipesModule {
}
PipesModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: PipesModule });
PipesModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function PipesModule_Factory(t) { return new (t || PipesModule)(); }, imports: [[
            TrustHtmlModule,
            Nl2BrPipeModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(PipesModule, { imports: function () { return [TrustHtmlModule,
        Nl2BrPipeModule]; }, exports: function () { return [TrustHtmlPipe, Nl2BrPipe]; } }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(PipesModule, [{
        type: NgModule,
        args: [{
                declarations: [],
                imports: [
                    TrustHtmlModule,
                    Nl2BrPipeModule
                ],
                exports: [TrustHtmlPipe, Nl2BrPipe]
            }]
    }], null, null); })();

export { PipesModule };
